salt/_pillar/metalk8s_nodes.py

```diff
@@ -40,6 +40,28 @@ def node_info(node, ca_minion):
     if node["metadata"]["name"] == ca_minion:
         roles.add("ca")
 
+    # Introduced by https://scality.atlassian.net/browse/ARTESCA-10079
```

Comment:
Do not link artesca ticket in metalk8s repo please :)

Reply:
Removed !!!